### PR TITLE
[WIP] Add tests for function verifyTradeInputValid in network contract

### PR DESCRIPTION
**Pull request by namnm1991**

### CODE REVIEW

File: test/sol5/kyberNetwork.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ const maxDestAmt = new BN(2).pow(new BN(255));
 const minConversionRate = new BN(0);
 const oneEth = new BN(10).pow(ethDecimals);
 const defaultNetworkFeeBps = new BN(25);
+const KYBER_MAX_QTY = new BN(10).pow(new BN(28)); // 10B tokens
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: please take this value from helper.js.
add it to line 17.

File: test/sol5/kyberNetwork.js [hunk 2]
```diff
@@ -1091,6 +1092,148 @@ contract('KyberNetwork', function(accounts) {
...
+    
+        it("test can not trade when src === dst", async () => {
+            await expectRevert(
+                network.tradeWithHintAndFee(networkProxy, srcToken.address, srcQty, srcToken.address, taker,
+                    maxDestAmt, minConversionRate, platformWallet, platformFee, emptyHint),
+                "src = dest"
+            );
+        });
+    
+        it("test can not trade when fee is to high", async () => {
```
Comment (ilanDoron) [inline test/sol5/kyberNetwork.js]: typo
to --> too

File: test/sol5/kyberNetwork.js (hunk 1 above)
Comment (namnm1991) [reply test/sol5/kyberNetwork.js]: resolved in 8a85cb2840731bfd6de26d589980f04b587331c4

File: test/sol5/kyberNetwork.js (hunk 2 above)
Comment (namnm1991) [reply test/sol5/kyberNetwork.js]: resolved in 8a85cb2840731bfd6de26d589980f04b587331c4

**Pull request merged**